packages/syft/src/syft/service/request/request.py

```diff
@@ -420,7 +421,7 @@ def _repr_html_(self) -> Any:
 
             """
 
-    def _coll_repr_(self):
+    def _coll_repr_(self) -> Dict[str, Union[str, Dict[str, str]]]:
```

Comment:
should just be
```suggestion
    def _coll_repr_(self) -> Dict[str, str]:
```
right?

Reply:
No @kiendang since the `"Status"` key of the returned dictionary is `status_badge` which is another dictionary

Reply:
I see.